submission/notifications/src/main/kotlin/ebi/ac/uk/notifications/api/RtClient.kt

```diff
@@ -0,0 +1,43 @@
+package ebi.ac.uk.notifications.api
+
+import arrow.core.getOrElse
+import ebi.ac.uk.notifications.exception.InvalidResponseException
+import ebi.ac.uk.notifications.exception.InvalidTicketIdException
+import ebi.ac.uk.notifications.integration.RtConfig
+import ebi.ac.uk.util.collections.second
+import ebi.ac.uk.util.regex.match
+import ebi.ac.uk.util.regex.secondGroup
+import org.springframework.util.LinkedMultiValueMap
+import org.springframework.web.client.RestTemplate
+import org.springframework.web.client.postForEntity
+import org.springframework.web.util.UriComponentsBuilder
+
+class RtClient(private val rtConfig: RtConfig) {
+    fun createTicket(content: String): String {
```

Comment:
I was expecting to a user email, how the notification will be sent to the end-user?